docs/sources/reference/run.md

```diff
@@ -587,6 +588,21 @@ division of CPU shares:
     101    {C1}		1	100% of CPU1
     102    {C1}		2	100% of CPU2
 
+### CPU period constraint
+
+The default period of CPUs is 100ms. We can use `--cpu-period` to set the 
+period of CPUs to limit containers's CPU usage. And usually `--cpu-period`
+should work with `--cpu-quota`.
+
+Examples:
+
+    $ docker run -ti --cpu-period=100000 --cpu-quota=50000 ubuntu:14.04 /bin/bash
+
+If there is 1 CPU, this means the container can get 50% worth run-time of 1 CPU
+every 100ms.
```

Comment:
mention that the `--cpu-period=100000` is the default, or use a non-default and explain it?